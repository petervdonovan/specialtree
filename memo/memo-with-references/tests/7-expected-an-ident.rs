use memo_with_references::memo;

#[memo(123)]
fn memome(cache: &'a str) {}

fn main() {}
