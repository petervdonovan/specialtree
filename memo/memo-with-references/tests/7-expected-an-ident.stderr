error: expected an ident
 --> tests/7-expected-an-ident.rs:3:8
  |
3 | #[memo(123)]
  |        ^^^
